internal/js/modules/k6/browser/tests/webvital_test.go

```diff
@@ -140,6 +146,9 @@ func TestWebVitalMetricNoInteraction(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, resp)
 
+	// Add a small delay to ensure all metrics are collected
+	time.Sleep(1 * time.Second)
```

Comment:
Do you mean samples collection here? If so, can we do it without the use of time so we can have a more deterministic test, please? We can close a `samplesDone` chan when the dedicated goroutine exits.